entregas/martinezDiego/reto2/Ppt.java

```diff
@@ -0,0 +1,35 @@
+package entregas.martinezDiego.reto2;
+
+import java.util.Scanner;
+
+class Ppt {
+    public static void main(String[] args) {
+
+        final int PIEDRA = 1;
+        final int PAPEL = 2;
+        final int TIJERA = 3;
+
+        int numeroComputadora = (int) (Math.random() * (TIJERA - PIEDRA + 1)) + PIEDRA;
+
+        System.out.println("Elige: 1 para Piedra, 2 para Papel, 3 para Tijera.");
+
+        int eleccionJugador = new Scanner(System.in).nextInt();
```

Comment:
Ok si es una única vez que se pregunta algo. Si no, entonces compensa hacer:

`Scanner entrada = new Scanner(System.in);`

Y luego

`int eleccionJugador = entrada.nextInt();`